src/ConnectionCard/BadgesContainer.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import * as React from 'react';
+import { ScrollView } from 'react-native';
+import StyleSheet from '../PlatformStyleSheet';
+
+type Props = {|
+  children: React.Node,
```

Comment:
pls add `read-only` for prop types https://flow.org/en/docs/types/interfaces/#toc-interface-property-variance-read-only-and-write-only